### PR TITLE
reporting page

### PR DESCRIPTION
**Pull request by orbeckst**
- COC violation reporting
- ombudspersons

### CODE REVIEW

**Comment by orbeckst** [comment]
@jennaswa and @IAlibay can I list your discord handles for people to DM you? If yes, can you please tell me which one to choose?

I also need to link it from the README.

If you have any other suggestions what should go in there please add/comment.

**Comment by IAlibay** [comment]
> @jennaswa and @IAlibay can I list your discord handles for people to DM you? If yes, can you please tell me which one to choose?
> 
> I also need to link it from the README.
> 
> If you have any other suggestions what should go in there please add/comment.

Yes please! I think highspeedmode is my discord name?

And thanks, will review a bit later.

**Review by IAlibay — APPROVED** [review]

File: REPORTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+feel right, you can contact one of the following people:
+
+- Oliver Beckstein (at the workshop)
+  - email: <orbeckst@mdanalysis.org> 
+  - discord: `@orbeckst`
+  - phone (no text messages): +1 480 727 9765
+  
+- Jenna Swarthout Goddard (online)
+  - email: <community@mdanalysis.org>
+  - discord: `@jennaswa`
```
Comment (orbeckst) [inline REPORTING.md:34]: @jennaswa I added your personal discord handle — is this ok?

**Comment by orbeckst** [comment]
I'll merge this for now – if necessary we change the live page.

**Pull request merged**